docs/specification.md

```diff
@@ -769,6 +773,25 @@ unit
     - The return amount for each voters is equal to or less than the voter's frozen tokens.
 - The lost of frozen tokens is due to the fact that the administrator has the right to `burn` or `transfer` frozen tokens of any proposers or voters.
 - If proposal is accepted, decision lambda is called.
+- Fails with `NOT_ADMIN` if the sender is not the administrator.
```

Comment:
Oh, that's somewhat unexpected to me. Why don't we let anyone call `flush`? Is there any harm in letting anyone call it? Ideally it should be called automatically when voting period ends, but since it's not possible, we have to explicitly call it. It doesn't seem to use `SENDER` value for anything. And if a non-admin user calls it before the admin, it will be even better.

It's unrelated to this issue, but if you agree with me, I will create an issue about removing this check or you can do it right in this PR because the change should be small I guess.

Reply:
I'm good with removing the admin restriction.